Makefile

```diff
@@ -165,8 +171,8 @@ openmatch-uninstall-local:
 # uninstall open-match
 .PHONY: openmatch-uninstall
 openmatch-uninstall:
-	kubectl delete psp,clusterrole,clusterrolebinding --selector=release=$(OM_NS)
-	kubectl delete namespace $(OM_NS)
+	helm uninstall -n ${OM_NS} ${OM_NS}
```

Comment:
Hi @govargo, 

Would you add `helm repo remove` instruction or new `make` commands for `helm repo add` and `helm repo remove` ?

Reply:
Added `helm repo remove` to Makefile.

Reply:
LGTM